### DISCUSSION TITLE
Show chat on screen during Live Streams

### DISCUSSION THREAD

**Discussion by BOLT04** [EddieHubCommunity/support#728] [Ideas] [answered]
Sometimes links get shared or cool questions are asked in chat, but this is only available during the stream. What does everyone think about Eddie adding a like section on the screen with the chat?

I was thinking about this while watching the video of one of the live streams and felt it would be cool to see chat 😃. Eddie highlights some, so let me know if anyone thinks this is a good idea.

**Comment by stemount** [comment, marked as answer]
@BOLT04 @eddiejaoude I am no YouTube pro, but on twitch, this actually is realtime chat, on YouTube there are varying levels of latency between the stream so sometimes chat could get horribly out of date (at the streamers end)

I do think you have a nice suggestion though, for example Eddie could bring up the chat as he is reading out shoutouts on a half and half of the screen scene or something?!

**Reply by BOLT04** [reply]
@stemount yeah the latency might be a problem. But what about enabling the chat box for the video?
This is what I've seen on some live streams from [.NET Foundation](https://www.youtube.com/watch?v=efvl_wtYSw8&t=39s):

![imagem](https://user-images.githubusercontent.com/18630253/106058877-c3968480-60e9-11eb-9386-649a0e8ac3c6.png)

So maybe there is a setting to enable this on YouTube?

**Reply by eddiejaoude** [reply]
@stemount that is an option 👍 . When I used to use OBS I was able to bring it on the screen on the side, I can probably do the same with vMix too. But the live stream chat should still be available afterwards.

@BOLT04 can you share a link to one of my live streams that does not have the chat after the live stream?

**Reply by BOLT04** [reply]
ohh I checked now and they do @eddiejaoude 👍. Maybe it doesn't have it in the first couple of hours after uploading it.

Then this isn't an issue, thank you for the response @eddiejaoude 😃

**Reply by eddiejaoude** [reply]
Oh I see, it takes YouTube time to process the live stream afterwards. Thank you David for raising and having the discussion 👍
